packages/hardhat-core/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "hardhat",
+  "name": "@oasislabs/hardhat",
```

Comment:
wait, no. we should not be modifying hardhat. Pleaase make it so that the `@oasisprotocol/sapphire-hardhat` plugin intercepts the `hardhat node` command when starting up a node and use the oasis ethervm, and `hardhat test` to also use ethervm.